tests/test_harvest_bookmarks.py

```diff
@@ -632,9 +575,67 @@ def do_test(self, conn_id):
+                
                 if stream not in no_replication_key:
+
+                    stream_bookmark_keys = self.expected_replication_keys().get(stream, set())
+                    assert len(stream_bookmark_keys) == 1  # There shouldn't be a compound replication key
+
+                    stream_bookmark_key = next(iter(stream_bookmark_keys))
+                    target_value = first_max_bookmarks.get(
+                        stream, {None: None}).get(stream_bookmark_key)
+                    target_min_value = first_min_bookmarks.get(
+                        stream, {None: None}).get(stream_bookmark_key)
+                    final_bookmark = second_max_bookmarks.get(
+                        stream, {None: None}).get(stream_bookmark_key)
+
+                    try:
+                        # Attempt to parse the bookmark as a date
+                        if state_value:
+                            if isinstance(state_value, str):
+                                state_value = self.local_to_utc(parse(state_value))
+                            if isinstance(state_value, int):
+                                state_value = self.local_to_utc(dt.utcfromtimestamp(state_value))
+
+                        if target_value:
+                            if isinstance(target_value, str):
+                                target_value = self.local_to_utc(parse(target_value))
+                            if isinstance(target_value, int):
+                                target_value = self.local_to_utc(dt.utcfromtimestamp(target_value))
+
+                        if target_min_value:
+                            if isinstance(target_min_value, str):
+                                target_min_value = self.local_to_utc(parse(target_min_value))
+                            if isinstance(target_min_value, int):
+                                target_min_value = self.local_to_utc(
+                                    dt.utcfromtimestamp(target_min_value))
+
+                        if final_bookmark:
+                            if isinstance(final_bookmark, str):
+                                final_bookmark = self.local_to_utc(parse(final_bookmark))
+                            if isinstance(final_bookmark, int):
+                                final_bookmark = self.local_to_utc(
+                                    dt.utcfromtimestamp(final_bookmark))
+
+                        if final_state_value:
+                            if isinstance(final_state_value, str):
+                                final_state_value = self.local_to_utc(parse(final_state_value))
+                            if isinstance(final_state_value, int):
+                                final_state_value = self.local_to_utc(
+                                    dt.utcfromtimestamp(final_state_value))
+
+                    except (OverflowError, ValueError, TypeError):
+                        print("Bookmarks cannot be converted to dates, comparing values directly")
```

Comment:
Please change this with tap-tester LOGGER

Reply:
Replaced `print` with LOGGER.